osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
> Furthermore I would suggest counterclockwise definition of the points.

In OpenDRIVE there is no order foreseen, i.e. the tool would need to be able to calculate and swap the sequence. Is this needed by the sensor modellers?

> Then there is the question about the reference frame.

The contour is given in _relative_ coordinates to that reference point, isn't it?